challenges/day-02/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Day 02</title>
+    <style>
+        h1 {
+            width: 20%;
+            height: 20%;
+            margin: 20px;
+            padding: 5px 5px 5px 15px;
```

Comment:
When applying margin/padding, you generally want to have similar values for left/right. In this case, I would use the following values: `5px 15px`. That way the top/bottom are 5px and the left/right are both 15px. Keeps things look a lot more consistent. 

Looking over my actual code, I ended up using `padding: 15px 20px`... take a look at both and see which one you prefer.

Reply:
I attempted both sizes and went with the one you did just to keep the same code. I tried both and the 5 x 15 is too small. It depends on everything else as well I guess.